app/api/helpers/storage.py

```diff
@@ -160,7 +160,7 @@ def upload_local(uploaded_file, key, **kwargs):
     Uploads file locally. Base dir - static/media/
     """
     filename = secure_filename(uploaded_file.filename)
-    file_relative_path = 'static/media/' + key + '/' + generate_hash(key) + '/' + filename
+    file_relative_path = '/tickets' + '/' + key + '/' + filename
```

Comment:
You have changed this for all uploads, now everything will go inside /tickets folder

Reply:
Yeah, just realized that. I'll be making a new method which uploads tickets to the `generated` folder.

Reply:
Yeah just realized that. I'll be making a new method to upload tickets separately as it shouldn't be preset in `static/`

Reply:
`if key == UPLOAD_PATHS['pdf']['ticket_attendee']:
        file_relative_path = 'generated/tickets/' + key + '/' + filename
    else:
        file_relative_path = 'static/media/' + key + '/' + generate_hash(key) + '/' + filename`
I think using this would save a lot of redundant new methods.

Reply:
```
 if key == UPLOAD_PATHS['pdf']['ticket_attendee']:
        file_relative_path = 'generated/tickets/' + key + '/' + filename
    else:
        file_relative_path = 'static/media/' + key + '/' + generate_hash(key) + '/' + filename
```

Reply:
I guess this would save us from creating a whole new method just to upload tickets.

Reply:
Add an optional parameter in the original method

```python
def upload(blah, blah, upload_dir=original_upload_dir)
```

And override that in the method call

Reply:
@iamareebjamal Done.